### DISCUSSION TITLE
Different display of widgets in the console and in vscode

### DISCUSSION THREAD

**Discussion by contractmax** [Textualize/textual#5248] [Q&A] [answered]
I'm a beginner in textual and python. I'm running a simple example. Everything is fine in VScode, but when I run the code through the console, a question mark is displayed instead of a line
![Screenshot_3](https://github.com/user-attachments/assets/5e32cd30-fe8e-44cb-b72f-c30b1fdeca19)

`from textual.app import App, ComposeResult
from textual.widgets import Input

class InputApp(App):
    def compose(self) -> ComposeResult:
        yield Input(placeholder="First Name")
        yield Input(placeholder="Last Name")

if __name__ == "__main__":
    app = InputApp()
    app.run()`

**Comment by willmcgugan** [comment]
Try installing the updated Windows Terminal https://apps.microsoft.com/detail/9n0dx20hk701?hl=en-US&gl=US

**Comment by contractmax** [comment]
Installed, but not working yet
![Screenshot_5](https://github.com/user-attachments/assets/28dad996-88ff-4cc3-8dae-fb1ae26e0399)

**Reply by TomJGooding** [reply, marked as answer]
That screenshot doesn't look like the Windows Terminal? Ensure you're launching Windows Terminal rather than Powershell -  you might also want to [change your default terminal application ](https://learn.microsoft.com/en-us/windows/terminal/install#set-your-default-terminal-application).

**Comment by contractmax** [comment]
I think I figured it out
![Screenshot_6](https://github.com/user-attachments/assets/34f027e0-27ad-4e40-a7f2-35550a404f2b)

before that I ran through the first
![Screenshot_7](https://github.com/user-attachments/assets/ec77f70a-f89b-4689-9ebf-fb770e9f7db6)

Thanks for your help
